### ISSUE TITLE
v0.5.0-alpha, TextDiffContent HTML shows texts in inappropriate style

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/materialstore#207]
See the screenshot

<img width="1440" alt="スクリーンショット 2022-04-03 18 12 51" src="https://user-images.githubusercontent.com/4467417/161420570-0cbf7a03-27bd-4577-b966-6465f120433e.png">

v0.5.0-alpha, TextDiffContent HTML is wrongly styled.

**Comment by kazurayam** [comment]
I know what caused this.

https://github.com/kazurayam/materialstore/blob/develop/src/main/resources/com/kazurayam/materialstore/reduce/differ/style.css

```
td, th {
    font-size: 12px;
    border-right: 1px solid #ccc;
    display: table-cell;
    overflow-wrap: break-word;
}
```

The CSS property `overflow-wrap: break-word` caused this.

**Comment by kazurayam** [comment]
There is another issue.

<img width="1552" alt="スクリーンショット 2022-04-03 18 28 56" src="https://user-images.githubusercontent.com/4467417/161421045-2846615f-9f13-46f0-aa47-e3e97fb39cca.png">

v0.4.x, the a sequence of `<span>` tags are compact --- has fewer number of white spaces in betweeen.
```
          <tr>
            <th class='code-equal'>1</th>
            <td class='code-equal'>
              <span class='blob-code-inner'><span class='blob-code-inner'><span class='unchanged'>@font-face {</span>
            </span>
          </span>
        </td>
```

v0.5.0-alpha, the sequence of`<span>` tags are delemited by more of whitespaces.
```
           <tr>
            <th class="code-equal">1</th>
            <td class="code-equal">
              <span class="blob-code-inner">
                <span class="blob-code-inner">
                  <span class="unchanged">@font-face {</span>
                </span>
              </span>
            </td>         
```

The whitespaces could be significant for CSS styling

**Comment by kazurayam** [comment]
fixed at v0.5.0
